README.md

```diff
@@ -134,6 +134,15 @@ used.
 * If you think a previous commit broke your feature, use [git's builtin debugging tools](https://git-scm.com/book/en/v2/Git-Tools-Debugging-with-Git)
 * (Note: currently aspirational): If you fix a bug, **write a test for it**.
 
+## Testing
+We use [Jest](https://jestjs.io/) for unit testing, and [Cypress](https://www.cypress.io/) for end-to-end testing.
+### Cypress
+* To run Cypress tests locally, first run `yarn ea-start-testing-db`, then in a separate terminal run either `yarn ea-cypress-run` for a CLI version, or `yarn ea-cypress-open` for a GUI version. To run specific tests in the CLI, you can use the `-s <glob-file-pattern>` option.
```

Comment:
You're gonna want to submit this upstream, so consider having LW-compatible text here.

Reply:
Hmm, that might be hard though. Maybe your LW-compatible text just has TODOs in it.

Reply:
Yeah, my plan was to write LW-specific text here in the upstream branch, since commands elsewhere in the readme are also repo-specific. I suppose that has the downsides of being a bit less maintainable should we need to modify this line later, and also it's one more thing I need to remember when writing the upstream PR--are there other downsides I'm not thinking of?